README_INTERFACE_TESTS.md

```diff
@@ -0,0 +1,134 @@
+# Interface tests
+
+Charm relation interfaces are information exchange protocols for Juju charms.
+In this repository, under each interface specification, one can find a list of interface tests. Their goal is to provide an as strict as possible set of rules to determine whether a given charm satisfies that interface protocol.
```

Comment:
"In this repository, under each interface specification, one can find a list of interface tests." is not exactly True